docroot/sites/default/services.local.yml

```diff
@@ -1,7 +1,11 @@
-# Uncomment the lines below to enable debugging.
 # Do not commit the changes on this file.
+# Uncomment lines to apply to your local environment.
 
-#twig.config:
-#  debug: true
-#  auto_reload: true
-#  cache: false
+#parameters:
+  # Uncomment the line below to resolve 500 error for edit pages: https://www.drupal.org/project/paragraphs/issues/3197720
+#  http.response.debug_cacheability_headers: false
```

Comment:
I see this fixed was reviewed and merged in https://github.com/massgov/openmass/pull/1037, tested, and confirmed that it works.